buildenv/jenkins/common/variables-functions

```diff
@@ -378,6 +368,26 @@ def set_build_variables() {
     echo "Using BUILD_ENV_CMD = ${BUILD_ENV_CMD}, BUILD_ENV_VARS_LIST = ${BUILD_ENV_VARS_LIST.toString()}"
 }
 
+def set_sdk_variables() {
+    SDK_FILENAME = params.SDK_FILENAME
+    if (!SDK_FILENAME) {
+        try{
+            SDK_PREFIX = "OpenJ9-JDK${SDK_VERSION}-PR${ghprbPullId}-${SPEC}-"
```

Comment:
We don't archive PRs anymore so we might as well remove this while we're here. Can likely merge Prefix and Suffix into the SDK_FILENAME declaration

Reply:
Changes made in latest push